src/client/components/chart/line_chart/view.tsx

```diff
@@ -0,0 +1,69 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+import { Component } from 'react'
+
+import { Renderer } from '../../../render2d/renderer'
+import { ChartModel } from '../model'
+
+import { LineChartController } from './controller'
+import { LineChartModel } from './model'
+import * as style from './style.css'
+import { LineChartViewModel } from './view_model'
+
+export type LineChartProps = {
+  model: ChartModel
+}
+
+@observer
+export class LineChart extends Component<LineChartProps> {
+  render() {
+    const chartModel = this.props.model
+    const lineChartModel = LineChartModel.of(this.props.model)
+    const viewModel = LineChartViewModel.of(lineChartModel)
+    const controller = LineChartController.of(lineChartModel)
```

Comment:
The controller and model are dependencies that should be passed as props, not created in a `render` call.

Pushing a partial fix which moves the controller to the `install.tsx` and model up a level.

What is the expected lifetime of a `LineChartModel`? Shouldn't it be created at the same time as the `ChartModel`?

Reply:
Moved up `install.tsx` to match lifetime of `ChartModel`.

1be6469576e57c25f097c411af4efd694a284fa5